src/main/java/io/redlink/more/data/repository/StudyRepository.java

```diff
@@ -169,8 +165,8 @@ public Optional<ApiRoutingInfo> getApiRoutingInfo(Long studyId, Integer observat
         }
     }
 
-    public Stream<ScheduleEvent> getObservationSchedule(Long studyId, Integer observationId) {
-        return jdbcTemplate.queryForStream(
+    public ScheduleEvent getObservationSchedule(Long studyId, Integer observationId) {
+        return jdbcTemplate.queryForObject(
```

Comment:
Attention: this will throw an exception if no result is found. You might want to catch it and return null or wrap it in an optional